persistent/Database/Persist/Class/PersistField.hs

```diff
@@ -44,6 +44,7 @@ import Data.Time (defaultTimeLocale)
 import Data.Time.Clock.POSIX (posixSecondsToUTCTime)
 #endif
 
+import Prelude
```

Comment:
Why import this? There is no `NoImplicitPrelude` pragma in there.

Reply:
It's a trick to get fewer warnings over time. By putting `Prelude` import last, you don't get a warning if you have `import Data.Semigroup (Semigroup(..))` above. This is nice for reducing warnings when compiling with different versions of `base` where more and more things are added to `Prelude.`

Reply:
Wow, didn't know that. I skimmed over this PR, looks fine. It'd be good to get this in, so I can build on it.